src/intelligence_layer/evaluation/aggregation/domain.py

```diff
@@ -41,6 +47,8 @@ class AggregationOverview(BaseModel, Generic[AggregatedEvaluation], frozen=True)
     crashed_during_evaluation_count: int
     description: str
     statistics: SerializeAsAny[AggregatedEvaluation]
+    labels: set[str] = set()
```

Comment:
are we sure that e.g. the `set` is created newly for each instance, or is it shared between instances and leads to unexpected behavior?

Reply:
They are not shared and the tests cover this